jest.config.ts

```diff
@@ -213,6 +213,7 @@ const config: Config.InitialOptions = {
     'static/app/**/*.{js,jsx,ts,tsx}',
     '!static/app/**/*.spec.{js,jsx,ts,tsx}',
   ],
+  prettierPath: null,
```

Comment:
I would discuss this with wider FE folks before mutating jest config - maybe drop an ask on discuss-frontend?

Reply:
good idea will do!